lib/ae_page_objects/core/dsl.rb

```diff
@@ -20,7 +20,9 @@ def element(name, options = {}, &block)
 
       self.element_attributes[name.to_sym] = klass
 
-      define_method name do |&block|
+      define_method name do |&block_for_define_method|
```

Comment:
@dtognazzini  the block passed to `element :some_element` is handled, how about the block passed to `some_element` method?  `some_element do { |e| e.text == xxx' }`